test/external-modules/die-with-dignity/build.gradle

```diff
@@ -14,17 +14,7 @@ GradleUtils.extendSourceSet(project, "main", "javaRestTest", tasks.named("javaRe
 
 tasks.named("javaRestTest").configure {
   it.onlyIf("snapshot build") { BuildParams.isSnapshotBuild() }
```

Comment:
Why can't these tests run against release builds?

Reply:
The test uses a test external-module, which are only bundled with snapshots. This is also why we must use the default distribution. We could probably rework this, but I'd like to not change that here.

Reply:
Nevermind, figured it out.